src/Listener/DoctrineMigrationDiffListener.php

```diff
@@ -18,6 +18,10 @@
 final class DoctrineMigrationDiffListener implements EventSubscriberInterface
 {
     private const UP_LINE = 'public function up(Schema $schema)';
+    private const MIGRATION_COMMANDS = [
+        'doctrine:migrations:diff',
+        'make:migration',
```

Comment:
can you add a test to ensure this command adds warning? 

You can see how it's done for `doctrine:migrations:diff'`

Reply:
done 🧪